element_deeplabcut/model.py

```diff
@@ -72,6 +73,8 @@ def activate(
         add_objects=_linking_module.__dict__,
     )
 
+    dlc_report.activate(f"{model_schema_name}_report")

```

Comment:
Should we mention this additional use of `model_schema_name` in the docstring?

Reply:
We don't have to. It will activated without any configuration.